### PR TITLE
Clear right panel after every command

### LINKED ISSUE #120: Bug - Information in Right Panel not Updated When Changes Occur (Clear/Edit...)
## Background
When we put the `clear` command, the tasks in the task list will be cleared. However, the information displayed on the right panel of a task is not cleared.
Similar bug. If I display information about a task and edit the task. The information in the right panel is not updated.

## Reproduce
Add a task. Click on the task to display information about the task on the right panel. Enter the `clear` command.
Second bug. Add a task. Click on the task to display information about the task on the right panel. Edit information about the task. Eg. `edit n/edited note`.  
image

## Information
Maybe you can look into the MVC design pattern in [Week 11 Topic](https://nus-cs2103-ay2324s1.github.io/website/schedule/week11/topics.html) where you can make it such that the Right Panel UI is notified every time the taskList changes.
image

## Acceptance
1. Clear the right panel information as well when the `clear` command is entered.
2. Updates the information in the Right Panel if you edit the task.

### PR DESCRIPTION
**Pull request by JCSnap**
close #120

### CODE REVIEW

**Review by asdfghjkxd — APPROVED** [review]
Looks good!

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 1]
```diff
@@ -124,6 +124,11 @@ void fillInnerParts() {
         commandBoxPlaceholder.getChildren().add(commandBox.getRoot());
     }
 
+    void clearTaskListPanel() {
```
Comment (asdfghjkxd) [inline src/main/java/seedu/address/ui/MainWindow.java:127]: One small nit, perhaps we could make this `private` instead?

**Review by asdfghjkxd — APPROVED** [review]
Looks good, except for one small nit!

**Review by seantehds — COMMENTED** [review]
LGTM Justin! Thanks for your work!

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 1 above)
Comment (JCSnap) [reply src/main/java/seedu/address/ui/MainWindow.java:127]: Sure!

**Pull request merged**